src/pages/ProfilePage.js

```diff
@@ -105,27 +105,15 @@ export default function ProfilePage() {
         <ExitButton />
       </Header>
       <main>
-        <ImgContainer min-width="280px" min-height="165px">
+        <ImgContainer>
           {image ? (
-            <UploadedImg
-              data-testid="inputImgUpload"
-              src={image}
-              alt=""
-              width="280px"
-              height="165px"
-            />
+            <UploadedImg data-testid="inputImgUpload" src={image} alt="" />
           ) : (
             <DefaultImg
               src={imageUploadInfo}
               alt="photoupload by clicking button with curved arrow right-hand-side, removing it by clicking the button with the bin on the left"
-              width="280px"
-              height="130px"
             />
           )}
-          <RemoveButton onClick={onImageRemove} data-testid="buttonRemove">
-            <span className="sr-only">Image remove button</span>
-            <img src={iconbin} alt="icon rotate arrow to the left" />
-          </RemoveButton>
           <input
             data-testid="inputImgUpload"
             id="imgUpload"
```

Comment:
hmmmh... you using 2x `data-testid="inputImgUpload"` at input and img...is it because input is hidden?

Reply:
Very good eyes, this has happened occasionally - i will check and change it